src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private ColorSupplier colSup = new ColorSupplier();
+
+    private Figure [] figure = {new Circle(new Random().nextInt(20) + 1, colSup.getRandomColor()),
+            new Square(new Random().nextInt(20) + 1, colSup.getRandomColor()),
+            new Triangle(new Random().nextInt(20) + 1, new Random().nextInt(20) + 1,
+                    colSup.getRandomColor()),
+            new Rectangle(new Random().nextInt(20) + 1, new Random().nextInt(20) + 1,
+                    colSup.getRandomColor()),
+            new Trapezoid(new Random().nextInt(20) + 1, new Random().nextInt(20) + 1,
+                    new Random().nextInt(20 + 1), colSup.getRandomColor())};
+
+    public Figure getRandomFigure() {
+        return figure [new Random().nextInt(figure.length)];
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle(10,"white");
```

Comment:
let's make 10 a constant https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#all-magic-numbers-in-your-code-should-be-constants